crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -396,7 +397,13 @@ define_tedge_config! {
                 #[tedge_config(example = "8001", default(value = 8001u16))]
                 port: u16,
             }
-        }
+        },
+
+        bridge: {
+            /// Set the bridge local clean session flag (required mosquitto >= 2.0.0)
+            #[tedge_config(example = "auto", default(value = false))]
+            include_localcleansession: AutoFlag,
```

Comment:
```suggestion
            include_localcleansession_disabled: AutoFlag,
```
To highlight the fact that we're going to set `localcleansession = false`

Reply:
This sounds really verbose.